Application/Frontend/components/ColorSchemeToggle/ColorSchemeToggle.tsx

```diff
@@ -4,6 +4,14 @@ interface IconProps {
   color: string;
 }
 
+/**
```

Comment:
Thank you for adding detail comment for our component of switching Dark/Light mode